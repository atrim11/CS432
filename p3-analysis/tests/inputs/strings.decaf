def int main() {
print_func("\"\"\"he\"lo world\n\t\"\t\"\t\t\n\n\"\"\"\"\"\"\"");
return 0;
}
