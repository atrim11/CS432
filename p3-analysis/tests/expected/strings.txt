Symbol 'print_func' undefined on line 2
